### DISCUSSION TITLE
getting started with development, error with make defconfig

### DISCUSSION THREAD

**Discussion by Pandagineer** [bitcraze/discussions#250] [Q&A]
Hello. I am trying to run this tutorial: https://www.bitcraze.io/documentation/tutorials/getting-started-with-development/

When I get to the step "make defconfig", I receive this message: "make: *** No rule to make target 'defconfig'.  Stop."

Please advise.

BTW, I noticed that when I open a terminal, I am in the "/home/bitcraze" directory, not in the /home/bitcraze/Desktop/projects/crazyflie-firmware/src/drivers/interface/ directory. But I've tried "make defconfig" from both locations, with no success.

**Comment by knmcguire** [comment]
Hi!

You are supposed to be in the root folder of the crazyflie-firmware repository, so on the VM that is /home/bitcraze/projects/crazyflie-firmware/

**Reply by knmcguire** [reply]
I changed the title of your discussions btw, to make it more descriptive to your problem

**Reply by Pandagineer** [reply]
Success. Thank you!

**Reply by knmcguire** [reply]
great! Please mark as answer if this indeed solved your problem
